README.md

```diff
@@ -40,12 +40,6 @@ If you are running Notebook 5.2 or earlier, enable the server extension by runni
 jupyter serverextension enable --sys-prefix jupyterlab_latex
```

Comment:
is this still the case? Provided we _are_ shipping the `jupyter_notebook_config.d/jupyterlab-latext.json` and `jupyter_server_config.d/jupyterlab-latex.json`, we need to also test that these things _work_ after being installed from the built sdist and whl

Reply:
Not the case with `notebook` 5.0.0 and `jupyterlab` 3.0.16 which I tested *both* when installing with `pip install .` and installing from wheel